app/models/queries/projects/selects/created_at.rb

```diff
@@ -32,8 +32,4 @@ class Queries::Projects::Selects::CreatedAt < Queries::Selects::Base
   def self.key
     :created_at
   end
-
-  def self.available?
-    User.current.admin?
-  end
```

Comment:
❓ How do you know, that this can be removed?

Reply:
In short: It was surprising to me that `created_at` was hidden behind admin permissions, while `updated_at` would be available to everyone. This felt inconsistent.

Neither @ulferts nor I could imagine why `created_at` should be hidden, so it was decided to consistently show both to everyone.